dephell/converters/setuppy.py

```diff
@@ -208,6 +208,8 @@ def dumps(self, reqs, project: RootDependency, content=None) -> str:
 
         # packages, package_data
         content.append(('packages', sorted(str(p) for p in project.package.packages)))
+        if project.package.package_dir:
```

Comment:
This is emitting `package_dir={'': ''},` unnecessarily...?

Reply:
Yes, this is just to keep output clearer.